app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
.present? thay vì .nil thì có đc không?

Reply:
Khi chưa đăng nhập thì current_user.nil? sẽ là nil, nil.nil? = true, mà nil.present? = false

Reply:
current_user.present?

Reply:
oh, You can remove exclamation mark to use present?

Reply:
đã sửa cmt lại ạ, nãy e ghi nhầm :))